Userland/Libraries/LibCore/LocalServer.cpp

```diff
@@ -50,28 +50,39 @@ LocalServer::~LocalServer()
         ::close(m_fd);
 }
 
-bool LocalServer::take_over_from_system_server()
+bool LocalServer::take_over_from_system_server(String const& socket_path)
```

Comment:
Normally this project uses west const style, but I never though much about it.
Now that I do, east const style does seem more logical and I think I'll adopt it too. :)

Reply:
That's awesome! East const makes more sense when you consider that C style types are read right-to-left, I think. I always write it as `char const *` on my own code. I try to follow the code styles of other projects in PRs, but I accidentally slipped that one in :^)